Fixed crash when there are trailing delimiters at the end of a line

2joy opened the pull request:
I used a piece of your code to fix the problem, you can see by comments. I needed to fix it to get https://github.com/lukasmartinelli/pgfutter works properly. I built the new version of pgfutter with fixed go-csv. After 2 week of every day usage there are no isues.

2joy committed:
Fixed crash when there are trailing delimiters at the end of a line

JensRantil commented:
Whoa! I'm happy this library is being used!

Thanks for the fix. Would it be possible for you to add a test case, too? If too much work, could you paste some example code/CSV here and I'll make sure to add a test case.

timotto committed:
feat(writer): introduce non-numeric & non-empty quote mode

JensRantil committed:
chore(ci): remove TravisCI

Travis CI was sunset in May 2021[1]. Will introduce Github Actions
instead.

[1] https://blog.travis-ci.com/2021-05-07-orgshutdown

JensRantil committed:
chore(ci): enable github actions

JensRantil committed:
chore(bors): reduce timeout

Defaults to one hour[1].

[1] https://bors.tech/documentation/getting-started/

bors[bot] committed:
Merge #22

22: Enable Github Actions r=JensRantil a=JensRantil



Co-authored-by: Jens Rantil <jens.rantil@normative.io>

bors[bot] committed:
Merge #19

19: feat(writer): introduce non-numeric & non-empty quote mode r=JensRantil a=JensRantil



Co-authored-by: Tim Otto <tim@timotto.io>

JensRantil committed:
chore(ci): delete branch after merge

Nice to have that done automagically.

bors[bot] committed:
Merge #23

23: chore(ci): delete branch after merge r=JensRantil a=JensRantil

Nice to have that done automagically.

Co-authored-by: Jens Rantil <jens.rantil@normative.io>

JensRantil committed:
chore(ci): run tests on all pushes

...not just pushes that are related to Bors.

bors[bot] committed:
Merge #24

24: chore(ci): run tests on all pushes r=JensRantil a=JensRantil

...not just pushes that are related to Bors.

bors merge

Co-authored-by: Jens Rantil <jens.rantil@normative.io>

JensRantil commented:
> Whoa! I'm happy this library is being used!
> 
> Thanks for the fix. Would it be possible for you to add a test case, too? If too much work, could you paste some example code/CSV here and I'll make sure to add a test case.

@2joy Ping. Spent a few minutes yesterday trying to reproduce this but was unable to. Do you think you could paste an example CSV file here?

JensRantil commented:
No response here. I'll close this.
